src/Util/Logger/AbstractFriendicaLogger.php

```diff
@@ -0,0 +1,158 @@
+<?php
+
+namespace Friendica\Util\Logger;
+
+use Friendica\Util\Introspection;
+use Friendica\Util\Strings;
+use Psr\Log\LoggerInterface;
+use Psr\Log\LogLevel;
+
+/**
+ * This class contains all necessary dependencies and calls for Friendica
+ * Every new Logger should extend this class and define, how addEntry() works
+ *
+ * Additional information for each Logger, who extends this class:
+ * - Introspection
+ * - UID for each call
+ * - Channel of the current call (i.e. index, worker, daemon, ...)
+ */
+abstract class AbstractFriendicaLogger implements LoggerInterface
```

Comment:
Why did you include `Friendica` in the class name? It already is in the `Friendica\Util\Logger` namespace.